src/_utils/DataStructureUtils.js

```diff
@@ -0,0 +1,8 @@
+export const twoWayMap = map => {
+    const twoWayMap = map;
+    Object.keys(map).forEach(k => {
```

Comment:
We'll be using 'reselect' for these types of conversions.
I also don't like our internal structure relying that much on what the API happens to return this day.